examples/qs-only/index.html

```diff
@@ -0,0 +1,54 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link rel="shortcut icon" href="favicon.png" type="image/x-icon" />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>Query suggestions only</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <div id="autocomplete"></div>
+
+      <div class="content">
```

Comment:
We can remove the whole `div` to keep the example minimal.

Reply:
Fixed.